friendsofphp/php-cs-fixer/2.19/manifest.json

```
@@ -0,0 +1,10 @@
+{
+    "aliases": ["cs-fixer", "php-cs-fixer"],
+    "copy-from-recipe": {
+        ".php-cs-fixer.dist.php": ".php-cs-fixer.dist.php"
+    },
+    "gitignore": [
+        "/.php-cs-fixer.php",
+        "/.php_cs.cache"
```

It's generated like that:
```suggestion
        "/.php-cs-fixer.cache"
```